docs/csharp/language-reference/operators/stackalloc.md

```diff
@@ -40,7 +40,7 @@ You can assign the result of the `stackalloc` operator to a variable of one of t
 
   In the case of pointer types, you can use a `stackalloc` expression only in a local variable declaration to initialize the variable.
 
-The content of the newly allocated memory is undefined. Starting with C# 7.3, you can use array initializer syntax to define the content of the newly allocated memory. The following example demonstrates various ways to do that:
+The content of the newly allocated memory is undefined. Beginning with C# 7.3, you can use array initializer syntax to define the content of the newly allocated memory. The following example demonstrates various ways to do that:
```

Comment:
Wondering why to change `Starting` to `Beginning`

Reply:
For consistency. There was a mix of both throughout the articles. I've picked up the "beginning" one.